src/components/Button/Button.jsx

```diff
@@ -0,0 +1,48 @@
+import * as S from './Button.styles';
+
+const Button = ({ type, color, handleClick, children }) => {
+  let background, border, text;
+
+  switch (color) {
+    case 'neutral':
+      background = 'transparent';
+      border = '#d4d4d4';
+      text = '#0063CC';
+      break;
+    case 'outline':
+      background = 'none';
+      text = '#0063CC';
+      border = '#0063CC';
+      break;
+    case 'destructive':
+      background = '#CC3300';
+      text = '#FEFEFE';
+      break;
+    case 'destructive text':
+      background = 'none';
+      text = '#CC3300';
+      border = '#53585D';
+      break;
+    case 'success':
+      background = '#66BB6A';
+      text = '#FEFEFE';
+      break;
+    default:
+      background = '#0063CC';
+      text = '#FEFEFE';
```

Comment:
please move all style logic in `.styles` file and create customized buttons according to styled-components docs and move colors to separate variables

Reply:
Fixed, I left only what is needed.